receiver/prometheusreceiver/README.md

```diff
@@ -94,6 +94,12 @@ receivers:
       interval: 30s
       collector_id: collector-1
 ```
+## Exemplars
+This receiver accepts exemplars coming in Prometheus format and converts it to Otlp format.
+1. Value is expected to be received in `float64` format
+2. Timestamp is expected to be received in `ms`
+3. Labels with key `span_id, spanid` in prometheus exemplars are set as Otlp `spanid` and labels with key `trace_id, traceid` are set as `traceid`
```

Comment:
Just to not leave out if sent using `traceid or spanid` keys. I've observed that few metrics are coming in with those keys as well.

Reply:
What would happen if both keys are present?

Reply:
@dashpole If both are present, the one that occurs last in the list of labels would be considered as `traceID` and the other one would be ignored

Reply:
I think i'd prefer only supporting `trace_id` to start.  We can't take away support for other keys after we've introduced it.  It also might be possible for "fixing" this to be done by a processor at some point.

One issue is that in the prometheus exporters, we convert the trace id back to the `trace_id` key.  If we accepted a `traceid` attribute as well, we would end up renaming the attribute to `trace_id` in the exporter and potentially confuse users.  If we only support `trace_id`, other trace id labels will still be round-tripped correctly from prometheus receiver to prometheus exporters.

Hopefully being consistent within OTel will encourage the community to align on `trace_id`.  Otherwise, we can revisit allowing other names in the future.

Reply:
@dashpole This makes a lot more sense. Thanks for the detailed explanation. Updated the code and pushed.

Reply:
I still see a few instances of traceid in the diff (at least in the documentation).  Can you update those as well?

Reply:
Done